AGA8CODE/JS/Example Of Implementation/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html><html lang="en"><head>
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <script src="p5.js"></script>
+    <title>Online Equation Of State : GERG 2008</title>  
+    <meta charset="utf-8">
+    <meta name="description" content="Equation Of State GERG 2008 running directly from your browser. Translation of the Eric Lemmon work from NIST">
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-WQBZC12KFX"></script>

```

Comment:
Can you please remove the google analytics?

Reply:
So sorry. Done.
Thanks for your patience.